WhaleSpotting/react-app/src/tests/components/Profile.test.tsx

```diff
@@ -51,6 +67,43 @@ test("When approval selected get data from API and change heading to Your Approv
     expect(title).toBeInTheDocument();
 });
 
+test("When click next page approvals load new records", () => {
+    jest.mock("../../api/apiClient", () => ({
+        __esModule: true,
+        fetchPendingSightings: jest.fn(async (pageNumber: number): Promise<SightingApiModel[]> => {
+            return Promise.resolve([mockexample1]);
```

Comment:
```suggestion
            switch (pageNumber) {
                case 1:
                    return Promise.resolve([mockexample1]);
                case 2:
                    return Promise.resolve([mockexample2]);
                default:
                    return Promise.resolve([]);
            }
```
If you want to check the data changes between pages this is how you would

Reply:
test amended

Reply:
changed the tests